cypress_shared/pages/page.ts

```diff
@@ -170,7 +174,7 @@ export default abstract class Page extends Component {
   }
 
   clickBreadCrumbUp(): void {
-    cy.get('li.govuk-breadcrumbs__list-item:nth-last-child(2)').click()
+    Page.clickBreadCrumbUp()
```

Comment:
Is there an advantage to having this method call a Page method, rather than doing the operation directly?